packages/documentation/src/stories/components/icon/icon.stories.tsx

```diff
@@ -263,6 +299,11 @@ const RotateTemplate = (args: Args) => (
 );
 
 export const Rotate: Story = RotateTemplate.bind({});
```

Comment:
The `name` property is now working for all stories except this one, not sure why...